.github/labeler.yml

```diff
@@ -0,0 +1,26 @@
+# Label to be added when the PR is updated:
+"Awaiting Review":
```

Comment:
> I picked this approach because it makes the set-up simpler, and I think re-adding "Awaiting Review" when a PR Is modified makes sense, but I can also see the argument for leaving all "Reviewed" => "Awaiting Review" changes manual. If you'd prefer, I can modify the labeler so that it only runs when the PR is first opened. Or I can try running two labeler actions, one to add "Awaiting Review" (which only runs when the PR is first opened), and one to add/remove the rest of the labels (which runs every time the PR is updated).

I do want to keep the distinction between "Coder Has Actioned Review" (someone's already reviewed this, should be prioritized, some level of commitment from the past reviewer) vs "Awaiting Review" (no one touched it yet). So if it's possible to run 2 actions, we can have one for sticking "Awaiting Review" on PRs when first opened, and the other for switching "Reviewed" => "Coder Has Actioned Review".

I'd wait for @sarken to take a look, though.

(If it's too annoying though, it's fine if we leave all the review status labels alone.)

Reply:
Splitting it into "Awaiting Review" when it's first opened and "Coder Has Actioned Review" when it's updated means that if someone updates the PR immediately after submitting it (say, to make changes suggested by Hound, but before a reviewer has looked at it), it'll be marked as "Coder Has Actioned Review," which doesn't sound like what you want.

It sounds like I should just have it add "Awaiting Review" at the start and otherwise ignore review labels.

Reply:
It turns out that I can make it so that it only adds "Coder Has Actioned Review" if it's been previously marked as "Reviewed: Action Needed." Alternatively, there might be a way to make it add "Coder Has Actioned Review" if someone has used Github's built-in system to request a review, although I'm having trouble testing that because I can't figure out how to request a review from myself. Do either of those sound like they might be appealing options?

Reply:
We don't really use GitHub's built-in review requests consistently, I'd say the first one.

You can request reviews from me if you need to test things out.

Reply:
Okay, I split the label action into three steps, and I updated the text of the PR to explain the intended behavior in detail.

(I didn't bother playing around with review requests because it sounds like that's not a feature you're interested in.)